src/views/PurchaseOrder.vue

```diff
@@ -198,6 +207,9 @@ export default defineComponent({
           this.mappingName = fieldMapping.mappingPrefName; 
         }
       },
+      areAllFieldsSelected() {
```

Comment:
We could use it for other use cases in the page

Reply:
Yes sir.